### ISSUE TITLE
Add JAXB to the POM in order to support Java 11+

### ISSUE THREAD

**Issue by gdubya** [keeps/dbptk-developer#473]
Description: DBPTK cannot import SIARD files when using Java 14: 

```
2020-12-11 14:59:00.531 ERROR 40250 --- [nio-8080-exec-7] c.d.c.c.common.utils.AsyncCallbackUtils  : [0:0:0:0:0:0:0:1] AsyncCallback error - RESTException: Remote exception caused by GenericException: Could not import the metadata to the Solr instance. caused by ModuleException: Error loading JAXBContext caused by JAXBException: Implementation of JAXB-API has not been found on module path or classpath. caused by ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory
```

Steps required to reproduce the bug:

```
➜java -version
openjdk version "14.0.2" 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12-46)
OpenJDK 64-Bit Server VM (build 14.0.2+12-46, mixed mode, sharing)
```

1.  Run the application with Java 14
2. Try to import a new SIARD file

This can be resolved by including the JAXB runtime in the project, since it is no longer a standard part of the JRE.

**Comment by luis100** [comment]
@gdubya Have you tested this fix on DBPTK Developer in Java 14?

**Comment by gdubya** [comment]
Yes, using java14 throws the error mentioned above. Switching back to Java8 works fine, but Java 8 is old.

**Comment by luis100** [comment]
Generally, we recommend using Java 8. The support for Java 11 and 14 is part of the plans, but we need to maintain Java 8 compatibility as this tool is sometimes used in old servers. Adding support to Java 11+ we need to ensure all drivers still work fine with it and all features are still functioning correctly. So, more than to add a PR we need tests to be done to ensure adding this dependency doesn't break something and that the tool actually works well in Java 11+ just by adding this dependency.

If you are up for the task please go ahead, if not we will need to wait until this becomes a necessity and some institution sponsors this development.
